stdlib/src/collections/optional.mojo

```diff
@@ -285,9 +285,9 @@ struct Optional[T: CollectionElement](
         """
         var output = String()
         var writer = output._unsafe_to_formatter()
-        write_to(writer, "Optional(")
+        writer.write("Optional(")
         self.format_to(writer)
-        write_to(writer, ")")
+        writer.write(")")
```

Comment:
**Suggestion** We could condense this down to a single line:

```suggestion
        writer.write("Optional(", self, ")")
```

Reply:
It errors out, we need conditional conformance for it.